### ISSUE TITLE
FP S4653: parameter names in scss starting with digit

### ISSUE THREAD

**Issue by LaurenceBarry** [SonarSource/sonar-css#199]
In the following scss, SonarCSS is parsing parameter names for ‘Unexpected unknown unit’:

```css
$ee-icons: (
      1gb                 : "\e06e",
      3g                  : "\e06d",
      3gb                 : "\e06c",
      4g                  : "\e06b",
      4gee                : "\e06a",
      4gee-resumptive     : "\e069",
      4gee-streaming      : "\e068",
```

For ‘4gee-streaming’ it reports: Unexpected unknown unit “gee-streaming”

**Comment by vilchik-elena** [comment]
Should be fixed with https://github.com/stylelint/stylelint/issues/4375
